mvn-build-all.bat

```diff
@@ -12,17 +12,28 @@ if not "%*" == "" (
 
 echo [INFO] Start a build.
 
+echo [DEBUG] Command arguments : "%DEFAULT_GOALS%"
+
+call mvn -U -f terasoluna-gfw-parent\pom.xml %DEFAULT_GOALS%
```

Comment:
- [ ] Is it really ``terasoluna-gfw-parent``? I think it should be root ``pom.xml``.